src/applications/accreditation/21a/components/06-background-information-chapter/NotIncludedViolationsList.jsx

```diff
@@ -5,8 +5,8 @@ const NotIncludedViolationsList = () => (
     <li>traffic fines of $300 or less</li>
     <li>any violation of law committed before your 16th birthday</li>
     <li>
-      any conviction for which the record was expunged under federal or state
-      law
+      any conviction for which the record was expunged under Federal or state
+      law.
```

Comment:
Another minor grammar point: the use of a period here is inconsistent with the other list items in this PR.